googlepeople.json

```diff
@@ -10,13 +10,13 @@
     "python_version": "3",
```

Comment:
Changes in unreleased.md file are missing. Could you please add changelog there?

Reply:
Added